src/cl_sii/libs/pydantic_utils.py

```diff
@@ -0,0 +1,5 @@
+PYDANTIC_VALUE_ERROR_URL = 'https://errors.pydantic.dev/2.4/v/value_error'
+
+PYDANTIC_STR_TYPE_ERROR_URL = 'https://errors.pydantic.dev/2.4/v/string_type'
+
+PYDANTIC_STR_TOO_SHORT_ERROR_URL = 'https://errors.pydantic.dev/2.4/v/string_too_short'
```

Comment:
If we pass `include_url=False` to `pydantic.ValidationError.errors()`, we won't need any of these. See https://docs.pydantic.dev/2.4/api/pydantic_core/#pydantic_core._pydantic_core.ValidationError.errors.